x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_privileges.ts

```diff
@@ -25,7 +25,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     this.tags(['failsOnMKI']);
 
     before(async () => {
-      await PageObjects.svlCommonPage.loginAsViewer();
+      await PageObjects.svlCommonPage.loginAsAdmin();
```

Comment:
Why do we need this change?

Reply:
🤔 Basically this will mean that only `admin` role in serverless can see our page?
I think we should check this with @gbamparop, @flash1293 and @LucaWintergerst.

Reply:
could you then add the privilege to a non admin in our tests? I'm afraid that tiding everything to an admin role will make us miss stuff in the future.